pkg/graph/resolvers/system_intake_actions.go

```diff
@@ -236,3 +239,175 @@ func CreateSystemIntakeActionRequestEdits(
 	}
 	return intake, nil
 }
+
+// CreateSystemIntakeActionReopenRequest reopens an intake request
+func CreateSystemIntakeActionReopenRequest(
+	ctx context.Context,
+	store *storage.Store,
+	fetchUserInfo func(context.Context, string) (*models.UserInfo, error),
+	input model.SystemIntakeReopenRequestInput,
+) (*models.SystemIntake, error) {
+	adminTakingAction, err := fetchUserInfo(ctx, appcontext.Principal(ctx).ID())
+	if err != nil {
+		return nil, err
+	}
+	intake, err := store.FetchSystemIntakeByID(ctx, input.SystemIntakeID)
+	if err != nil {
+		return nil, err
+	}
+
+	if intake.State == models.SystemIntakeStateOPEN {
+		return nil, &apperrors.BadRequestError{
+			Err: fmt.Errorf("request is already open"),
```

Comment:
I noticed this myself and it's actually a problem with the error type: https://github.com/CMSgov/easi-app/blob/main/pkg/apperrors/errors.go#L215

It's worth changing to some different verbiage if you have any suggestions.

Reply:
Thanks Lee! Either way, I'd maybe just update that error to something like `Invalid Request` or maybe request could not be understood. Probably would say the system intake is already open instead of request.

Reply:
Addressed in [4ca06aa](https://github.com/CMSgov/easi-app/pull/2162/commits/4ca06aaddd85306a5387de3bfc0b997911e6b4f5)